interface-elements-for-web/articles/tree-list/filtering/search-panel.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Search Panel
+author: Anna Kondratova
+---
+# Search Panel
+
+Use the Search Panel to locate data and highlight search result by typing the filter criterion in the search box.
+
+![SearchPanel](../../../images/img17905.png)
+
+## Applying the Search Panel Filter Criterion
+
+Press the ENTER key or click the **Search** button to apply a filter criterion typed in the search panel. Otherwise, a filter is automatically applied in 1.2 seconds.
```

Comment:
"the ENTER key"
Is this the correct key capitalization?
Microsoft Manual of Style seems to state not using all caps:
https://www.microsoftpressstore.com/articles/article.aspx?p=2228449&seqNum=12

Reply:
Modified.